AGENTS.md

```
@@ -82,789 +72,217 @@ Each API interface must be organized in separate directories:
+### Control API - RFC9457 Errors
 
-# 5) Open docs/UI
-open http://localhost:8000/docs     # Swagger UI
-open http://localhost:8000/redoc    # ReDoc
-open http://localhost:5173          # SvelteKit Frontend
+All Control API errors must return `application/problem+json`:
 
-# 6) Full CI checks (heavy - only before PR)
-just ci-check
+```python
+{
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)